### ISSUE TITLE
386BSD 0.0 dist.fs boot floppy causes VM to reboot

### ISSUE THREAD

**Issue by planetbrian** [86Box/86Box#2936]
**Describe the bug**
When attempting to boot the 386BSD 0.0 dist.fs 1.44MB floppy from oldlinux.org, it will not work. I have read the other two issues regarding a triple fault (not sure what that is), but I would still like to know if there is any leeway in resolving this bug.

**Expected behavior**
Well, it should boot to this (screenshot from gunkies.org). Anything else regarding the disk probably would work fine, as I have tested other UNIX ports around this era.
![image](https://user-images.githubusercontent.com/99565026/205485547-de636e7f-75f8-4e91-9b7b-c1f1886a777d.png)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - 86Box version: 3.11, x64, ODR

**Comment by OBattler** [comment]
This is a duplicate of this older issue: https://github.com/86Box/86Box/issues/530 .

I have spent a lot of time debugging this and I have found out that it does strange Interrupt acceses. Basically, it does an instruction that results in a divide by zero, but the IDT (Interrupt Descriptor Table) has not been initialized yet, causing the triple fault reset.

I'm closing this - the older issue can be used to further discuss this.
